foundations/installations/installation_overview.md

```diff
@@ -1,14 +1,14 @@
 ### Introduction
 
-The first step for building any website is having the right tools. For us, that means setting up a development environment for writing good code.
+The first step in building any website is having the right tools. For us, that means setting up a development environment for writing good code.
 
 Many online development courses use in-browser code editors or "sandboxes," which give you the tools and programs needed to accomplish the task at hand and nothing else. You'll use some of these sandboxes throughout the early stages of The Odin Project since they're great for getting started quickly. However, the best way to set yourself up for long-term success is to operate in a real development environment.
 
 We won't lie to you: installing packages, editors, and even entire operating systems can be very frustrating. However, having the experience of setting up a development environment to run the code you'll write is an invaluable, real-world skill you'll carry with you for the rest of your career.
 
 ### The installation plan
```

Comment:
Let's make the lesson template linter happy here and make this `### Lesson overview`

Reply:
> > Let's make the lesson template linter happy here and make this `### Lesson overview`
> 
> That heading doesn’t only contain an overview of the tasks to be done, so I think changing that to Lesson Overview will create some inconsistency. However, you can do the changes necessary if you see fit.

We have a linter in place that enforces a certain template. You will see the errors shown at the bottom of this PR.

![image](https://github.com/TheOdinProject/curriculum/assets/61510135/23fcc4ad-f8ef-486d-a8e4-d47f4582ab12)


I do not make changes on other peoples' PRs. If you want this merged, please implement the changes I'm requesting.

Reply:
It brought up another lint error.

Reply:
I'll ignore this one - thanks.